cli/tools/coverage.rs

```diff
@@ -221,8 +231,34 @@ impl PrettyCoverageReporter {
         println!("{}", colors::red(&line_coverage));
       }
 
+      let output_lines =
+        if let Some(original_source) = maybe_original_source.as_ref() {
+          original_source.split('\n').collect::<Vec<_>>()
```

Comment:
Does this handle CRLF files correctly?

Reply:
Right, windows.

I believe so, we also do the same in program_state (https://github.com/denoland/deno/blob/master/cli/program_state.rs#L324).